api_get.py

```diff
@@ -0,0 +1,66 @@
+from sanic import Sanic
+from sanic import response
+from serializers import *
+from functions import *
+from sanic.response import text
+import requests
+
+app = Sanic('some_name')
+
+
+# GET
+
+@app.route('/')
+async def get_handler(request):
+    return response.json({'message': 'Hello world!'}, headers={'Service': 'SDA'}, status=200)
+
+
+@app.route('/payment')
+async def get_all(request):
+    payment_rows = await get_all_payments()
+    result = payments_schema.dump(payment_rows)
```

Comment:
Please, let's follow next flow:
1. We need to get data from database
2. Convert data to json object and return it as it is

In case of request from the outside the flow would be like:
1. Get request data
2. Validate via marshmallow
3. Do some logic

Reply:
changed.